its/src/test/java/com/sonar/it/shared/TestUtils.java

```diff
@@ -296,6 +281,16 @@ public static void deleteLocalCache() {
     }
   }
 
+  // Populate the plugin cache before the tests to avoid race conditions in the begin step
+  public static void warmupLocalCache(Orchestrator orch, TemporaryFolder temp) throws IOException {
+    var projectName = "warmupLocalCache";
```

Comment:
There's no practical reason to have the stupidCase
```suggestion
    var projectName = "WarmUpLocalCache";
```